package.json

```diff
@@ -6,22 +6,17 @@
   "scripts": {
     "start": "parallelshell \"node web/index.js\" \"npm run watch:css\" \"npm run watch:js\"",
     "test": "npm run test:server && npm run test:browser && npm run lint",
-
```

Comment:
I think this happened when I ran `npm install react-ga --save`

I'm guessing this is going to be a recurring issue if we want to keep them formatted like this?